app.py

```diff
@@ -11,12 +11,10 @@
 
 # Setup docs settings
 docs_settings = {
-    "docs_url": None,
-    "redoc_url": "/docs",
     "openapi_prefix": settings["DOCS_URL_PREFIX"],
```

Comment:
I'm not sure this settings is useful, I fell like it is used to customize where the docs will look for `openapi.json` ([which will be served at the root by default for fastapi](https://fastapi.tiangolo.com/advanced/extending-openapi/#the-normal-process)).

Maybe it also have a wrong name, as I understand it it may not affect the url of documentation endpoints?

Reply:
Swagger and Redoc clients will indeed try to fetch the schema via `/openapi.json` (using an absolute path). 
But Idunn is typically deployed under `/maps/detail` prefix. And the clients would fail to fetch the schema file using this absolute path.
With `DOCS_URL_PREFIX=/maps/detail`, docs will be available via `/maps/detail/docs` and the clients will fetch `openapi.json` correctly from `/maps/detail/openapi.json`

Do you thing this setting name should be changed to be more explicit ?

Reply:
Oh no you're right, I assumed it would use a relative path, this makes more sense.

About the name I'm not sure anything I could think of would really be better, I guess this is fine :)